lib/WasmReader/WasmBinaryReader.h

```diff
@@ -94,6 +94,7 @@ namespace Wasm
             virtual WasmOp ReadFromBlock() override;
             virtual WasmOp ReadFromCall() override;
             virtual WasmOp ReadExpr() override;
+            virtual void Unread() override { m_pc--; m_funcState.count--; }
```

Comment:
Agreed with this concern. If keeping this function, maybe rename to `Backtrack`?